tox.ini

```
@@ -98,7 +98,7 @@ deps =
   pre-commit>=1.20.0
 skip_install = true
 commands =
-  python -m pre_commit run {posargs:--all-files}
+  python -m pre_commit run -v {posargs:--all-files}
```

Keep it overridable.
```suggestion
  python -m pre_commit run {posargs:--all-files -v}
```